### PR TITLE
Remove quote

### PR DESCRIPTION
**Pull request by ambatlle**
* scripts/alterDeleteFlagToQuotes.sql must be executed previously to run remove quote new comment, it creates a new necessary column to mark the quotes as removed
* commands/main.go adds the new rmquote command invocation
* commands/quotes.go adds the controller method to remove quote
* database/quotes.go adds thow methods, FindQuoteById to find a quote by it's id, MarkQuoteAsRemoved which updates the removed column with current timestamp to mark the quote as erased.

### CODE REVIEW

**Comment by ambatlle** [comment]
to execute sql script

cat scripts/alterDeleteFlagToQuotes.sql | docker exec -i diosteama_db_1 psql -U <user>

File: commands/main.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ func Command(update tgbotapi.Update, bot *tgbotapi.BotAPI) {
 		quienes(update, bot, args)
 	case "es":
 		es(update, bot, args)
-
+	case "rmquote":
```
Comment (javipolo) [inline commands/main.go:38]: Use delquote better, for historical reasons (it was the command in the irc bot)

**Review by javipolo — CHANGES_REQUESTED** [review]

File: commands/main.go (hunk 1 above)
Comment (ambatlle) [reply commands/main.go]: Change done, not tested yet

**Comment by itorres** [comment]
Todos estos cambios están dentro de #33 por lo que cerramos esta PR.

**Pull request closed without merging**